src/codegen_call_thread.cpp

```
@@ -92,474 +92,418 @@ static std::unordered_set<std::string> collectReferencedVars(const LambdaExpr &l
+    requireArgs(e, 1);
+
+    llvm::Value *envPtr = nullptr;
+    llvm::Function *thunk = nullptr;
+
+    // Case 1: inline lambda expression
+    auto *lambda = std::get_if<std::unique_ptr<LambdaExpr>>(&e.args[0]->data);
+    if (lambda) {
+        LambdaExpr &lam = **lambda;
+
+        auto referencedVars = collectReferencedVars(lam);
+        std::vector<std::pair<std::string, llvm::AllocaInst*>> captures;
```

Cppcheck (2.20.0)

[style] The function 'operatorSymbol' is never used.

(unusedFunction)